playwright.config.ts

```diff
@@ -70,7 +70,8 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: "npm run dev",
+    // Longterm this needs to test the actual deployment.
+    command: process.env.CI ? "npm run build && npm run start" : "npm run dev",
```

Comment:
npm run start runs like prod, so faster. Only run npm run dev when running tests locally for debugging.